panel/widgets/tqdm_indicator.py

```diff
@@ -0,0 +1,139 @@
+# I did not put it into the indicators.py file yet as it raises errors due to circular imports I
+# believe
+import panel as pn
+import param
+from tqdm.asyncio import tqdm as _tqdm
```

Comment:
Panel uses relative imports, never us an absolute import like this (this is probably also the source of your circular import issues).

```suggestion
import param

from tqdm.asyncio import tqdm as _tqdm

from ..viewable import Viewer
```

Reply:
Thanks using this and an extended relative import `..pane.markup` I was able to move `Tqdm` to the indicators file.